frontend/packages/operator-lifecycle-manager/src/types.ts

```diff
@@ -257,6 +257,10 @@ export type PackageManifestKind = {
           required?: APIServiceDefinition[];
         };
       };
+      entries: {
```

Comment:
I think this should fix the TS errors you're seeing in CI
```suggestion
      entries?: {
```